pubspec.yaml

```diff
@@ -15,15 +15,15 @@ dev_dependencies:
   test: ^1.24.3
 
 dependencies:
-  cbor: ^5.1.2
+  cbor: ^6.1.0
```

Comment:
The update to this dependency is currently blocked by `dart_dcaf`. This should be resolved as soon as a new `dart_dcaf` version has been released.

Reply:
Thanks to @falko17, this has now been resolved :)